npms/skale-cool-socket/.eslintrc.js

```diff
@@ -97,6 +97,9 @@ module.exports = {
         "no-use-before-define": "off", // [ "error", { "variables": false,  "functions": false } ],
         "lines-between-class-members": [ "error", "never" ],
         "no-var": "error",
-        "no-unused-vars": "error"
+        "no-unused-vars": "error",
```

Comment:
Why don't use the config from a parent folder?

Reply:
Yes, you are right. One is enough.